README.md

```diff
@@ -174,25 +171,26 @@ where `credential` is an ECDAA credential obtained earlier and `prng` is an `ecd
 Notice that the signature thus created is not TPM-specific.
 This means that verification of a TPM-generated signature proceeds as usual, using `ecdaa_signature_FP256BN_verify`.
 
-## Random number generator
+## Cryptographically-secure random numbers
 
 Many of the functions provided by this library (particularly, those used by an Issuer or a Member)
-require a pseudo random number generator (type `ecdaa_prng`).
-The security of these algorithms depends critically on the proper seeding of this prng.
-This means that the first use of any `ecdaa_prng` MUST be preceeded by a call to
-`ecdaa_prng_init` (or `ecdaa_prng_init_custom`, see below) on the prng.
+require a source of cryptographically-secure random numbers.
 
-In `ecdaa_prng_init`, the seed for the `ecdaa_prng` is generated from Libsodium's
-`randombytes_buf` function.
-A discussion on how this function works and any caveats can be found at Libsodium's webpage.
+Such functions take a parameter of type `ecdaa_randomness_func_t`,
```

Comment:
Can that name (and the associated header file name) be shortened?  Any reason to prefer `randomness` to `rand` or `random`?

Somewhat related, what's the reason for the `_t` suffix?  Would just `ecdaa_rand_func` suffice?  Some of the names get really lengthy and make it hard to fit even a one-parameter function call on one line.

Reply:
Yea, good call. I have a tendency to get really verbose with my naming. `ecdaa_rand_func` in the file `rand.h` works for me.